athlinks-ontology.ttl

```diff
@@ -6,542 +12,381 @@
+	owl:versionIRI <http://adp.athlinks.com/schema/1.0-3.2/> ;
+  # import our owlized version of schema.org
+	owl:imports <http://adp.athlinks.com/schemax/> ;
+	rdfs:comment "This file contains Athlinks data vocabulary. The vocabulary is based on the schema.org ontology with our own extensions for concepts specific to human endurance events."@en ;
+	rdfs:label "Athlinks data vocabulary"@en ;
+	dc:contributor "astn" , 
+                 "nextdude" , 
+                 "warren" ,
+                 "berg" .
+
+####
+# Properties
+####
+:eventAction a owl:ObjectProperty ;
+	rdfs:subPropertyOf owl:topObjectProperty ;
+	rdfs:domain :AthleteResult ;
+	rdfs:range :AthleticEventAction ;
+	rdfs:comment "A pointer to the the athletic event action that is being ranked."@en ;
+	rdfs:label "exercise action"@en .
+
+:legacyData a owl:ObjectProperty ;
+	rdfs:domain owl:Thing ;
+	rdfs:range :LegacyDataRole ;
+	rdfs:comment "A legacy data record belonging to another system that is related to this object."@en ;
+	rdfs:label "legacy data"@en .
+
+# TODO: This shouldn't be defined on schema:Person. It should be computed by query.
+# Or available in racer metrics. - nextdude
+:participatedIn a owl:ObjectProperty ;
+	rdfs:domain schema:Person ;
+	rdfs:comment "An indication of a person participating in an ActivityType."@en ;
+	rdfs:label "participated in"@en .
 
-###  http://adp.athlinks.com/schema/legacyData
-:legacyData rdf:type owl:ObjectProperty ;
-            rdfs:domain owl:Thing ;
-            rdfs:range :LegacyDataRole ;
-            rdfs:comment "A legacy data record belonging to another system that is related to this object."@en ;
-            rdfs:label "legacy data"@en .
+# virtual property - computed on the fly by the api
+:age a owl:DatatypeProperty ;
+	rdfs:domain schema:Person ;
+	rdfs:range xsd:integer ;
+	rdfs:comment "The age (in years) of a person. Note that we don't store this constantly changing value in our database, but rather compute it dynamically when queried based on schema:birthDate."@en ;
+	rdfs:label "age"@en .
 
+:rank a owl:DatatypeProperty ;
+	rdfs:subPropertyOf schema:position ;
+	rdfs:domain :AthleteResult ;
+	rdfs:range [
+          rdf:type owl:Class;
+          owl:unionOf (
+            xsd:integer
+            xsd:string
+          );
+        ];
+  rdfs:comment "Rank of a course time result in the context of a particular athlete cohort. The rank may differ from its [position](http://schema.org/position), which is assumed to be sequential. Two items with sequential positions can share the same rank."@en ;
+  rdfs:label "rank"@en .
 
-#################################################################
-#    Data properties
-#################################################################
+:siblingOrdering a owl:DatatypeProperty ;
+	rdfs:domain owl:Thing ;
+	rdfs:range xsd:integer ;
+	rdfs:comment "The numeric order of an object among siblings of the same property or type."@en ;
+	rdfs:label "sibling ordering"@en .
```

Comment:
It might avoid some confusion later if we use some kind of list structure to store the ordering instead. 
I'm imagining the case where you have the same Subject part of multiple orderings. Having the ordering defined outside of the subject and closer to the parent that cares about the ordering could help this.
For example.. using [rdfs containerMembershipProperty](https://www.w3.org/TR/rdf-schema/#ch_containermembershipproperty)

```
_:raceStartOrdering a rdf:Seq;
                   rdf:_1 _:startGroupPro;
                   rdf:_2 _:startGroupElite;
                   rdf:_3 _:startGroupGeneral.
```

Using a [list](https://www.w3.org/TR/rdf-schema/#ch_list)
```
_:raceStartOrdering a rdf:List;
              rdf:first _:startGroupPro;
              rdf:rest [ a rdf:List; rdf:first _:startGroupElite;
                            rdf:rest [ a rdf:List; rdf:first _:startGroupElite;
                                          rdf:rest [ a rdf:List; rdf:first _:startGroupRest;
                                                       rdf:rest rdf:nil];
                                        ];
                            ].
```

Reply:
There is also sugar for rdf:List in Turtle [sugar](https://www.w3.org/TR/turtle/#collections)